assets/tests/components/ladder.test.tsx

```diff
@@ -538,9 +548,10 @@ describe("ladder", () => {
     const mockDispatch = jest.fn()
 
     const timepoints = ["t0", "t1", "t2"]
-    const incomingGhost: Ghost = {
+    const incomingGhost: Ghost = ({
       id: "ghost-incoming-y0622",
-    } as Ghost
+      blockWaivers: [],
+    } as unknown) as Ghost
```

Comment:
Same question as in `routeLadder.test.tsx`. Why is the `as unknown` needed now?

Reply:
I noticed you fixed the `as unknown` in `routeLadder.test.tsx`. Looks like there's one more here and one more in `routeLadder.test.tsx` (line 605).